react/CHANGELOG.md

```
@@ -8,6 +8,23 @@
 
 
+### Bug Fixes
+
+* avoid first message flicker ([#93](https://github.com/tambo-ai/tambo/issues/93)) ([87c78d3](https://github.com/tambo-ai/tambo/commit/87c78d3b7569d3b8f385aecdf0aef3487b70697c))
+
+
+### Miscellaneous
+
+* **main:** release react 0.13.0 ([#96](https://github.com/tambo-ai/tambo/issues/96)) ([631aa4b](https://github.com/tambo-ai/tambo/commit/631aa4be495947297acdd24ba2c2c2f929066e29))
+
+## [0.13.0](https://github.com/tambo-ai/tambo/compare/react-v0.12.1...react-v0.13.0) (2025-03-11)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)